### PR TITLE
Adding r-metaskat

### PR DESCRIPTION
**Pull request by matuskosut**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by matuskosut** [comment]
@conda-forge/staged-recipes ready for review

**Comment by matuskosut** [comment]
@cbrueffer could you maybe help with review & merge of this R package?

File: recipes/r-metaskat/meta.yaml [hunk 1]
```diff
@@ -36,8 +36,10 @@ requirements:
   run:
     - r-base
     - r-skat
-    - libstdcxx-ng                     # [not win]
-    - libgcc-ng                        # [not win]
+    - clang_osx-64                     # [osx]
+    - clangxx_osx-64                   # [osx]
+    - gcc_linux-64                     # [linux64]
+    - gxx_linux-64                     # [linux64]
```
Comment (cbrueffer) [inline recipes/r-metaskat/meta.yaml:42]: Are these dependencies really necessary? I'd think they're already pulled in by the compiler bits below.

File: recipes/r-metaskat/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-skat
+  run:
+    - r-base
+    - r-skat
+    - {{ compiler('c') }}              # [not win]
```
Comment (cbrueffer) [inline recipes/r-metaskat/meta.yaml:39]: Actually, none of the run entries except the `r-` and `gcc-libs` lines should be needed; should have looked closer the first time.

Comment (matuskosut) [reply recipes/r-metaskat/meta.yaml]: oh yeah, you are right, I now tested locally and pushing an update

**Comment by cbrueffer** [comment]
Looks good!

**Comment by matuskosut** [comment]
thanks!

**Pull request merged**